.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,26 @@
 Specify library name and version:  **lib/1.0**
 
-<!-- This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks! -->
+Changes tested in the following configurations:
+<!--
+It's not mandatory to test every possible configuration if it's not easy to do so,
+but we'd appreciate it if you let us know beforehand which ones you're sure do work.
+If you don't have access to a specific platform to test your changes on,
+feel free to reflect that in the validate() method.
+
+Also, please specify specific configuration information if necessary
+-->
+ - [ ] Linux:
+ - [ ] macOS:
+ - [ ] Windows:
+ - [ ] Other, please specify:
+
+<!--
+This is also a good place to share with all of us **why you are submitting this PR**
+(specially if it is a new addition to ConanCenter):
+Is it a dependency of other libraries you want to package?
+Are you the author of the library?
+Thanks!
+-->
```

Comment:
Does this part appear in the PR markdown? If this is hidden then I am happy to remove it as well as the preceding part.

Reply:
I think it's visible if you click `edit`